internal/integration/integration_test.go

```diff
@@ -116,11 +137,25 @@ func TestInterfaceListField(t *testing.T) {
 
 	require.Len(t, resp.Beings, 3)
 
+	// We should get the following three beings:
```

Comment:
note to self: post-rebase, need to apply these changes to `TestInterfaceListPointerField`.